game_states.py

```diff
@@ -1,11 +1,45 @@
+import pygame
 import pymlgame
 import time
 
-import drawing
 from misc import Point
 
 
-class GameState():
+class PygameSurfaceDecorator:
```

Comment:
I think this is a neat workaround :) Do you think we'll utilize this one more often? We might give it it's own module then -- it's kind of unexpected here. Is it time for a `util.py` yet?